x-pack/distributions/internal/observabilitySRE/qa/acceptance/docker/elasticsearch/config/elasticsearch-fips.yml

```diff
@@ -0,0 +1,15 @@
+# Elasticsearch settings
+discovery.type: single-node
+http.port: 9200
+network.host: 0.0.0.0
+# Security settings
```

Comment:
Does `xpack.security.fips_mode.enabled` need to be set to `true`, or is this not required with the the `elasticsearch-cloud-ess-fips` docker image?

Reply:
(https://www.elastic.co/docs/deploy-manage/security/fips-140-2)

Reply:
I will add this as investigation in https://github.com/elastic/ingest-dev/issues/5320 (added a point in google doc to track this down).

Reply:
As promised https://github.com/elastic/ingest-dev/issues/5320#issuecomment-3084618426